### ISSUE TITLE
OS X: multicore rendering ineffective

### ISSUE THREAD

**Issue by jeanlain** [ValveSoftware/portal2#198]
This is not specific to the beta, but in my experience, the "multicore rendering" option never had any affect on portal 2 CPU usage and frame rate. In other source games (except CS:GO), especially after the linux update, multicore rendering offers a significant speed boost and I thought that the portal 2 beta would offer similar benefits, but no dice.
Enabling the option or setting "mat_queue_mode" to 2 does nothing. CPU usage remains around 130%, which is typical of other source games with multicore rendering disabled. From time to time however, CPU usages increases to 220+ % for unclear reasons, but the frame rate is not affected (this also happens in CS:GO).

Tested at lowest resolution on a quad Core 2006 Mac Pro, OS X 10.9.2.

**Comment by mariodonofrio** [comment]
You're right - multicore rendering has never been supported for Portal2 (or CSGO) on OSX. Hopefully this will change for CSGO as I've recently finished the work to enable that feature on that particular title (it might be a while before being made available to users). For Portal2 the Linux porting has changed/touched the OSX code as little as possible (to prevent new bugs as much as possible) so this feature will not have been added.

**Comment by PaulDixon** [comment]
can we beg for it to be added :D I got a mac mini as little media computer and it's a painful experience. Any extra help would make it a bit nicer to use :D - but I understand from a developers perspective if it ain't really broke....don't fix it.

**Comment by stevethepocket** [comment]
Hmm. I wonder if this is true for their older games as well; I noticed that when I play TF2 under Linux, my CPU fan never gets loud like it does in Windows. That would explain the inferior performance as well.

**Comment by jeanlain** [comment]
In my experience, other source games except CS:GO have good multicore support on OS X, at least on a quad-core machine. The performance increase in TF2 seems to be rather modest, for some reason. But in L4D2, it helps a lot on my Mac Pro that has old processors and a more recent GPU.
